app/src/main/java/com/iron/espresso/presentation/home/study/list/LocationFragment.kt

```diff
@@ -40,6 +45,27 @@ class LocationFragment : BaseFragment<FragmentLocationBinding>(R.layout.fragment
                 this.isRefreshing = false
             }
         }
+
+        scrollListener()
+    }
+
+    private fun scrollListener() {
+        binding.studyList.addOnScrollListener(
+            object : RecyclerView.OnScrollListener() {
+                override fun onScrolled(recyclerView: RecyclerView, dx: Int, dy: Int) {
+                    super.onScrolled(recyclerView, dx, dy)
+
+                    val linear =
```

Comment:
```suggestion
                    val layoutManager =
```
lm, layoutManager, linearLayoutManager 등 레이아웃 매니저임을 알 수 있는 네이밍이 어떨까 싶네용

Reply:
네이밍 수정하였습니다!